app/views/pages/get-started/build-your-first-app/removing-data-from-the-database.liquid

```diff
@@ -0,0 +1,140 @@
+---
+metadata:
+  title: Removing data from the database
```

Comment:
@z-x Is there already a page for adding data into database?

Reply:
@piotrze - yup, https://github.com/mdyd-dev/platformos-documentation/pull/1551/files#diff-f18a913956f70ba3fa8a3196645157dbca0b20d451e90da55b240844e3872074